certbot/CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 
-*
+* Error out in apache installer when mod_ssl is not available.
```

Comment:
nit: We previously errored out, just the error message wasn't very helpful.

What do you think about saying something like "Improved error message in apache installer when mod_ssl is not available." and moving this to the changed section?